metron-platform/metron-solr/metron-solr-common/src/test/java/org/apache/metron/solr/dao/SolrSearchDaoTest.java

```diff
@@ -510,5 +512,13 @@ public void getGroupResultsShouldProperlyReturnGroupResults() {
     assertNull(level2GroupResults.get(1).getGroupResults());
   }
 
-
+  private SolrDocument createSolrDocument(String sensorType, Long timestamp) {
```

Comment:
Thank you for this - simple types like this should not be mocked. Good improvement.